core/block_svg.ts

```diff
@@ -1135,6 +1135,9 @@ export class BlockSvg
       .getBlockStyle(blockStyleName);
     this.styleName_ = blockStyleName;
 
+    dom.removeClass(this.svgGroup_, this.styleName_)
+    dom.addClass(this.svgGroup_, blockStyleName)
```

Comment:
Could you write up how you tested this? Since `this.styleName_ = blockStyleName` is assigned on line 1136 this probably  isn't working how you expect it to.